charts/karpenter/Chart.yaml

```diff
@@ -14,3 +14,12 @@ home: https://karpenter.sh/
 icon: https://repository-images.githubusercontent.com/278480393/dab059c8-caa1-4b55-aaa7-3d30e47a5616
 sources:
   - https://github.com/aws/karpenter/
+dependencies:
+  - name: karpenter-crd
+    version: "v0.23.0"
```

Comment:
does this get auto-updated?

Reply:
Not at the moment but will do soon. Have to test that a bit.

Reply:
A few other things such as the go.mod need to be updated that's why it's better suited as a follow up: will most likely happen here https://github.com/aws/karpenter/blob/main/hack/release/common.sh#L53

Reply:
donen in https://github.com/aws/karpenter/pull/3331/commits/f55598213fd527bf131385d9c11f9e43298184e6